callBackPractice/callBackPractice.js

```diff
@@ -24,48 +24,93 @@ firstItem(foods, (firstItem) => {
   console.log(`The first item is ${firstItem}.`);
 });
 
+const firstItem = (foods, cb) => {
+		cb(foods[0]);
+};
+
 // Write a function called getLength that passes the length of the array into the callback
 
 getLength(foods, (length) => {
   console.log(`The length of the array is ${length}.`);
 });
 
+const getLength = (foods, cb) => {
+  for (let i = 0; i < foods.length; i++) {
+    cb(foods[i].length);
+  }
+};
+
 // Write a function called last which passes the last item of the array into the callback
 
 last(foods, (lastItem) => {
   console.log(`The last item in the array is ${lastItem}.`);
 });
 
+const last = (foods, cb) => {
+	cb(foods[foods.length-1]);
+};
 // Write a function called sumNums that adds two numbers and passes the result to the callback
 
 
 sumNums(5, 10, (sum) => {
   console.log(`The sum is ${sum}.`);
 });
 
+const sumNums = (num1, num2, cb) => {
+	cb(num1 + num2);
+};
+	
 // Write a function called multiplyNums that adds two numbers and passes the result to the callback
 
 multiplyNums(5, 10, (product) => {
   console.log(`The product is ${product}.`);
 });
 
+const multiplyNums = (num1, num2, cb) => {
+	cb(num1 * num2);
+};
+
 // Write a function called contains that checks if an item is present inside of the given array.
 // Pass true to the callback if it is, otherwise pass false
 
 contains(foods, 'ribeye', (result) => {
   console.log(result ? 'ribeye is in the array' : 'ribeye is not in the array');
 });
 
+const contains = (foods, x, cb) => {
+  for(let i = 0; i < foods.length; i++)
```

Comment:
So the goal here is to get your CB function to return a "True"/"False" value. Maybe try and finish this so that your code will run and execute as it's called on line 76